pages/one-page.jsx

```diff
@@ -16,6 +16,7 @@ module.exports = React.createClass({
         <Header locale={this.props.locales[0]} alt={this.getIntlMessage('donate_to_mozilla')}></Header>
         <SingleForm
           billingAddress={true}
+          privacyPolicyTest={this.getIntlMessage("privacy_policy_var_b")}
```

Comment:
Should we go further and delete the privacy policy test code within the SingleForm component so that every page going forward gets the new privacy policy?